Dockerfile

```diff
@@ -22,10 +13,18 @@ ENV VARNISH_VARNISHD_PARAMS ''
 ENV VARNISH_VARNISHNCSA_PARAMS ''
 ENV VARNISH_CACHE_TAGS_HEADER Cache-Tags
 
+COPY patches /tmp/patches/
+COPY config /tmp/config/
+COPY scripts /tmp/scripts/
+COPY healthcheck.sh /usr/bin/healthcheck.sh
+COPY startup.sh /usr/bin/startup.sh
```

Comment:
Let's standardize on using `/usr/local/bin/docker-entrypoint.sh` for the main entrypoint script.
Renamed the source file as well.